mage_ai/frontend/components/PipelineDetail/index.tsx

```diff
@@ -465,6 +516,20 @@ function PipelineDetail({
       ) {
         event.preventDefault();
         savePipelineContent();
+      } else if (onlyKeysPresent([KEY_CODE_A], keyMapping) || onlyKeysPresent([KEY_CODE_B], keyMapping)) {
+        if (!event.repeat && !isInputElement(event)) {
```

Comment:
We check if the an input / text area is focused directly rather than using `textareaFocused` because `textareaFocused` is not always accurate :'(